elasticsearch/v7/client.go

```diff
@@ -0,0 +1,109 @@
+package v7
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"strings"
+
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+	jsoniter "github.com/json-iterator/go"
+
+	"github.com/Trendyol/go-elasticsearch-connect-couchbase/config"
+
+	"github.com/elastic/go-elasticsearch/v7"
+)
+
+type Client interface {
+	Bulk(reader *bytes.Reader) error
+}
+
+type client struct {
+	v7Client *elasticsearch.Client
```

Comment:
I also want to write a unit test for Elastic v7 client but it's hard to mock. Do you have any ideas to mock elasticsearch.Client?

Reply:
In my opinion, no need to mock an elastic client. We don't have to write unit tests for this. We should write an integration test instead.

Reply:
make sense to me. I will add it 👍

Reply:
Is there any way to write an integration test without using 3rd library like test containers? otherwise, I need to add test containers.